code/game/gamemodes/wizard/artefact.dm

```diff
@@ -153,7 +153,7 @@
 	src.spawn_amt_left = spawn_amt
 	src.desc = desc
 
-	processing_objects.Add(src)
+	START_PROCESSING(SSobj, src)
```

Comment:
so this never gets removed from processing Nice.

Reply:
[It's eternal](https://www.youtube.com/watch?v=nlxfDvSyPKA)